src/backend/distributed/commands/citus_version.c

```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * citus_version.c
+ *	  Routines related to the version of the Citus binaries
+ *
+ * Copyright (c) 2012-2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "fmgr.h"
+
+#include "citus_version.h"
+#include "utils/builtins.h"
+
+
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(citus_running_version);
+
+#if !defined(CITUS_BINARY_VERSION)
```

Comment:
At the moment, we read major/minor/catalog version from citus.control file and I would like to keep this as it is, because people may forget to update these versions when they need to change the catalog version.

However patch version will only change during new releases, so it is OK if we hardcode it somewhere along with CITUS_BUILD_INFO. (We can add another item to your check list for releases to  ensure it is not forgotten)

I can add another .h file for those two variables and extend citus_running_version function to show patch version too. Does that sounds like a plan?

Apart from that, I am not sure how to populate CITUS_BUILD_INFO easily from build system. Will we need to do some regex match & replace?